test/relayserver/RegistrationManager.test.ts

```diff
@@ -3,15 +3,14 @@ import { HttpProvider } from 'web3-core'
 import { toBN } from 'web3-utils'
 import {
   ContractInteractor,
-  ServerConfigParams,
   constants,
   RelayHubConfiguration
 } from '@rsksmart/rif-relay-common'
 import { KeyManager } from '../../src/relayserver/KeyManager'
 import { RegistrationManager } from '../../src/relayserver/RegistrationManager'
 import { RelayServer } from '../../src/relayserver/RelayServer'
 import { ServerAction } from '../../src/relayserver/StoredTransaction'
-import { ServerDependencies } from '../../src/relayserver/ServerConfigParams'
+import {ServerConfigParams, ServerDependencies} from '../../src/relayserver/ServerConfigParams'
```

Comment:
missed linter here?

Reply:
Actually we don't have any linter checker here, that was introduced on #150, in this PR you have the rules but nothing that forces you to apply them. What i'm trying to say is that the change on the #150 will fix all these errors. I will apply these fixes here manually anyway.

Reply:
neat, thanks